src/js/components/FormField/doc.js

```diff
@@ -28,6 +28,23 @@ export const doc = FormField => {
     label: PropTypes.oneOfType([PropTypes.string, PropTypes.node]).description(
       'A short label describing the field',
     ),
+    name: PropTypes.string.description(
+      `The name of the value data when in a Form and the name of
+      the input field.`,
+    ),
+    pad: PropTypes.bool.description(
+      'Whether to add padding to align with the padding of TextInput.',
+    ),
+    required: PropTypes.bool.description('Whether the field is required.'),
+    validate: PropTypes.oneOfType([
+      PropTypes.object, // regular expression
+      PropTypes.func,
+    ]).description(
+      `Validation rule. Provide a regular expression or a function. If a
```

Comment:
can we expand the propType to be a shape: 
```
PropTypes.shape({ regexp: RegExp, message: PropTypes.string })
```

I don't think React has a RegExp proptype. maybe we can add "any" and explain that regexp should be a regexp instance ?

Reply:
done